int16 num_markers
FiducialMarker[] markers
int32 seq_num